src/models/Model.php

```diff
@@ -1,6 +1,7 @@
 <?hh // strict
 
 abstract class Model {
+
```

Comment:
could either make Cache a singleton, or add a protected static Cache (or just the Map/dict and remove the Cache class) here